src/com/facebook/buck/apple/clang/UmbrellaHeader.java

```diff
@@ -28,7 +28,13 @@ public UmbrellaHeader(String targetName, ImmutableList<String> headerNames) {
   }
 
   public String render() {
+    String swiftGeneratedHeader = String.format("%s-Swift.h", targetName);
```

Comment:
Let's move this to a constructor and have a set in case we need to add more stuff in future. Something like that:

```
ImmutableSet<String> headersToExclude;

public UmbrellaHeader(...) {
    ...
    headersToExclude = ImmutableSet.of(
        // Target-Swift.h header file should be excluded from the list of header names.
        // The umbrella header should not import the generated Objective-C header.
        // It is the job of the module map to make sure that Swift code is accessible
        // to Objective-C via the Target-Swift.h header file.
      String.format("%s-Swift.h", targetName));
}
```